schema/partner_artist.js

```diff
@@ -0,0 +1,70 @@
+import gravity from '../lib/loaders/gravity';
+import cached from './fields/cached';
+import Partner from './partner';
+import Artist from './artist/index';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLInt,
+  GraphQLBoolean,
+} from 'graphql';
+
+const PartnerArtistType = new GraphQLObjectType({
+  name: 'PartnerArtist',
+  fields: () => ({
+    cached,
+    id: {
+      type: GraphQLString,
+    },
+    artworks_count: {
+      type: GraphQLInt,
+    },
+    display_on_partner_profile: {
+      type: GraphQLBoolean,
+    },
+    published_artworks_count: {
+      type: GraphQLInt,
+    },
+    published_for_sale_artworks_count: {
+      type: GraphQLInt,
+    },
+    represented_by: {
+      type: GraphQLBoolean,
+    },
+    sortable_id: {
+      type: GraphQLString,
+    },
+    use_default_biography: {
+      type: GraphQLBoolean,
+    },
+    biography: {
+      type: GraphQLString,
+    },
+    partner: {
+      type: Partner.type,
+    },
+    artist: {
+      type: Artist.type,
+    },
+  }),
+});
```

Comment:
I'm still of the opinion that we should be nesting the counts inside of a `counts` key; then casting them as numeral types for easy formatting. And that any Boolean flags should be prefixed by `is_` for legibility in templates and other code.